daml-foundations/daml-tools/daml-extension/syntaxes/daml12.tmLanguage-update.xml

```diff
@@ -514,14 +511,22 @@
 			<string>#infix_op</string>
 		</dict>
 		<dict>
-			<key>begin</key>
+			<key>comment</key>
+			<string>We treat :: as cons. It's important this definition precede the rule for : (and that rule precede the one for arbitrary operators).</string>
+			<key>match</key>
 			<string>(::|∷)</string>
+			<key>name</key>
+			<string>keyword.operator.daml</string>
+      </dict>
+		<dict>
+			<key>begin</key>
+			<string>(:)</string>
 			<key>beginCaptures</key>
 			<dict>
 				<key>1</key>
 				<dict>
 					<key>name</key>
-					<string>keyword.other.double-colon.haskell</string>
+					<string>keyword.other.double-colon.daml</string>
```

Comment:
@akalburgi as we discussed, are we not going to change this (similarly line 481 at least)?